templates/terraform/examples/spanner_database_basic.tf.erb

```diff
@@ -6,8 +6,4 @@ resource "google_spanner_instance" "main" {
 resource "google_spanner_database" "database" {
   instance = google_spanner_instance.main.name
   name     = "<%= ctx[:vars]['database_name'] %>"
-  ddl = [
```

Comment:
We'll want to leave the example in there - the examples are how our tests work and we do need to test these fields.

Reply:
@ndmckinley I added another file for testing purposes with the DDL property. PTAL.

Reply:
Looks great!  Can you add it to the `terraform.yaml` file's list of examples, so it gets properly executed during testing?